### PR TITLE
[qt] Add transliteration enable/disable setting

### PR DESCRIPTION
**Pull request by biodranik**

### CODE REVIEW

File: map/framework.hpp [hunk 1]
```diff
@@ -669,7 +669,7 @@ class Framework : public PositionProvider,
 
 public:
   void AllowTransliteration(bool allowTranslit);
-  bool LoadTransliteration();
+  static bool LoadTransliteration();
```
Comment (Ferenc-) [inline map/framework.hpp:672]: Why make only this one static?

Comment (vng) [reply map/framework.hpp:672]: Transliteration (same as 3D for example) should be the state of Framework (inner drape engine).
The current "settings" implementation (not the best one) allows to make this (and some others) functions static, but I'd not do it. Tomorrow, they will be state modifiers.

Comment (Ferenc-) [reply map/framework.hpp:672]: Yep I know. And a bunch of other member functions could be static as well. But I don't think either that they should be. And also on the caller side, `LoadTransliteration` is called on an object.

Comment (biodranik) [reply map/framework.hpp]: Agree, removed static. @vng @Ferenc- PTAL

File: qt/preferences_dialog.cpp [hunk 2]
```diff
@@ -81,6 +81,16 @@ namespace qt
       });
     }
 
+    QCheckBox * transliterationCheckBox = new QCheckBox("Transliterate to Latin");
+    {
+      transliterationCheckBox->setChecked(framework.LoadTransliteration());
+      connect(transliterationCheckBox, &QCheckBox::stateChanged, [&framework](int i)
+      {
+        framework.SaveTransliteration(static_cast<bool>(i));
+        framework.AllowTransliteration(static_cast<bool>(i));
```
Comment (vng) [inline qt/preferences_dialog.cpp:90]: nit: I prefer explicit ```bool const enable = i > 0;```

**Review by vng — APPROVED** [review]

File: qt/preferences_dialog.cpp (hunk 2 above)
Comment (biodranik) [reply qt/preferences_dialog.cpp]: In other places, it's still static cast. For a one-liner it's clearer IMO.

**Pull request merged**